src/lang/std/engineConnection.ts

```diff
@@ -1899,15 +1899,10 @@ export class EngineCommandManager extends EventTarget {
     range: SourceRange,
     commandTypeToTarget: string
   ): string | undefined {
-    const values = Object.entries(this.artifactGraph)
-    for (const [id, data] of values) {
-      // // Our range selection seems to just select the cursor position, so either
-      // // of these can be right...
-      if (
-        (data.range[0] === range[0] || data.range[1] === range[1]) &&
-        data.type === commandTypeToTarget
-      )
-        return id
+    for (const [artifactId, artifact] of this.artifactGraph) {
+      if ('codeRef' in artifact && isOverlap(range, artifact.codeRef.range)) {
+        if (commandTypeToTarget === artifact.type) return artifactId
+      }
```

Comment:
this was a fix that was needed after the artifact graph migration that I missed. 

This not working basically meant that we were calling zoom_to_fit with an empty array instead of with the path/object id, so not that big of a deal because it still had the effect of making sure the user didn't end up inside of the extrusion after extruding.

It might seem like I shouldn't have fixed it because this PR means it's not called anymore, but that's temporary while we get a fix for "zoom_to_fit".